src/json.hpp

```diff
@@ -1235,16 +1235,16 @@ void from_json(const BasicJsonType& j, ArithmeticType& val)
     }
 }
 
-template<typename BasicJsonType, typename... Args>
-void from_json(const BasicJsonType& j, std::pair<Args...>& p)
+template<typename BasicJsonType, typename A1, typename A2>
+void from_json(const BasicJsonType& j, std::pair<A1, A2>& p)
 {
-    p = {j.at(0), j.at(1)};
+    p = {j.at(0).template get<A1>(), j.at(1).template get<A2>()};
```

Comment:
Er, you're right. I was talking about std::get here, which is not what you're doing at all. I mis-read it. What you have looks good!